mscr-ui/src/common/utils/has-permission.tsx

```diff
@@ -114,13 +125,54 @@ export function checkPermission({
     return true;
   }
 
+  console.log(rolesInTargetOrganizations);
+  // Return true if user has data model editor role in target organization
+  if (
+    rolesInTargetOrganizations?.includes('DATA_MODEL_EDITOR')||rolesInTargetOrganizations?.includes('ADMIN')
+  ) {
+    return true;
+  }
+  
+
   // Return true if user has admin role in target organization
   if (
-    rolesInOrganizations.includes('ADMIN') &&
+    rolesInOrganizations.includes('') &&
     rolesInTargetOrganizations?.includes('ADMIN')
   ) {
     return true;
   }
 
+  return false;
+}
+
+export function checkEditPermission({
+  user,
+  owner
+}: checkPermissionProps) { 
+  console.log("Checking editing permisison");
+  if (owner?.includes(user.id)) {
+    //user is the owner, Check for personal Contents
+    return true;
+  } else {
+    //Gruop Content
+    const rolesInOrganizations = Object.keys(user.organizationsInRole);
+    console.log(rolesInOrganizations);
+    const adminProp = 'ADMIN';
```

Comment:
The roles could be listed as an enum type in mscrUser interface, then they could be referred to as e.g. `Roles.Admin`. See Format or State interfaces for examples of enums.

Reply:
Done!!